rules/spec.yaml

```diff
@@ -0,0 +1,98 @@
+openapi: 3.0.0
+info:
+  description: The Rules Backend API
+  version: "1.0.0"
+  title: Rules Backend API
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+tags:
+  - name: rulesv1
+    description: Calls related to rules
+paths:
+  /api/v1/{tenant}/rules:
```

Comment:
For now I went with the approach to only have a single file just for the sake of simplicity. Although the Rules backend is storing the files in object storage in a structure that'll allow us to move to multiple files per tenant without breaking anything.

Also, if we allow multiple files, I think it'd be great to have one endpoint that returns all rules merged in a single response. This will make the rule syncer's work much easier.

Reply:
Sorry, I don't get that. The `$ref: '#/components/schemas/Rules'`  points that Rules are set of Groups, right? there is not file notion.

Reply:
Ah ok - I understand this now. The identifier for the set of rules is the rule group name - tenants can choose to create and update any they like while keeping some level of semantic separation if they wish,

Reply:
I'm actually a bit confused - as I understand this still means one file per request, but using the `Rules` schema, that uses `RuleGroups`? is this what we want?
We can also test here: https://app.swaggerhub.com/apis/observatorium/rules-backend/1.0.0#/Rules

Reply:
and here where we define yaml as content-type and the use of the `Rules` schema: https://github.com/observatorium/api/blob/44d2f6c58aebab68636f524b7ffdd5debcc304bd/rules/spec.yaml#L48-L53